souto-app/src/components/ticket.js

```diff
@@ -69,25 +69,26 @@ export default class Ticket extends Component {
             />
           </Grid>
           <Grid
+            zeroMinWidth
             item
             xs={this.props.isManager ? 8 : 10}
             style={styles.textBox}
             onClick={() => {
-              this.setState({ isOpen: true })
+              this.setState({ isOpen: true });
             }}
           >
-            <Typography style={styles.title}>
+            <Typography maxLength="2" style={styles.title}>
               {this.props.info.title}
             </Typography>
-            <Typography style={styles.rearInfo}>
+            <Typography noWrap style={styles.rearInfo}>
```

Comment:
And this?

Reply:
its neccessery so the content of the typography wont exceed the ticket borders